darts/tests/test_timeseries.py

```diff
@@ -810,6 +810,50 @@ def test_append_values(self):
         assert appended.time_index.equals(expected_idx)
         assert appended.components.equals(series.components)
 
+    def test_pre_and_append_values_empty(self):
```

Comment:
we should check it for datetime index as well (we can use pytest parametrization)